src/app/components/side-bar/side-bar.component.html

```diff
@@ -8,6 +8,12 @@ <h3>{{ 'MISC.BALANCE' | translate | uppercase }}</h3>
             {{helpers.prettyCoins(wallet.balance)}}
             <span class="balance-currency">Linda</span>
         </span>
+        <!-- Staking -->
```

Comment:
Current stake be integrated into the getBalance function like the masternode collateral is so the balance reflects the total balance